File: js/nova/database/ResultSet.js
var ResultSet = function () {
	var self = this;
	
	this.numCols = 0;
	this.rows = nova_null;
	
	this.ResultSet = function (rows, numCols) {
		self.this(rows, numCols);
	};
	
	this.destroy = function () {
	};
	
	this.this = function (rows, numCols) {
		self = rows;
		self = numCols;
	};
	
	this.accessor_numRows = function () {
		return count;
	};
	
	this.super = function () {
	};
	
	
};

File: java/nova/database/ResultSet.java
package nova.database;

import java.util.Optional;
import nova.exception.ExceptionData;
import nova.exception.Exception;
import nova.exception.DivideByZeroException;
import nova.io.NovaConsole;
import nova.primitive.number.Number;
import nova.primitive.number.NovaByte;
import nova.primitive.number.NovaShort;
import nova.primitive.number.NovaInt;
import nova.primitive.number.NovaLong;
import nova.primitive.number.NovaFloat;
import nova.primitive.number.NovaDouble;
import nova.primitive.Null;
import nova.primitive.number.Char;
import nova.primitive.Bool;
import nova.datastruct.list.NovaArray;
import nova.datastruct.list.IntArray;
import nova.datastruct.list.CharArray;
import nova.datastruct.list.DoubleArray;
import nova.datastruct.list.IntRange;
import nova.thread.Thread;
import nova.thread.async.Async;
import nova.gc.GC;
import nova.math.NovaMath;
import nova.NovaObject;
import nova.NovaString;
import nova.System;
import nova.Class;

public class ResultSet extends NovaObject
{
	
	public int numCols;
	public NovaArray rows;
	
	
	
	
	public ResultSet(NovaArray rows, int numCols)
	{
		init(rows, numCols);
	}
	
	public int accessor_numRows()
	{
		return rows.count;
	}
	
	public void init(NovaArray rows, int numCols)
	{
		this.rows = rows;
		this.numCols = numCols;
	}
	
}
